src/components/AvatarWithImagePicker.js

```diff
@@ -115,26 +216,45 @@ class AvatarWithImagePicker extends React.Component {
                     <AttachmentPicker>
                         {({openPicker}) => (
                             <>
-                                <Pressable
-                                    style={[styles.smallEditIcon, styles.smallAvatarEditIcon]}
-                                    onPress={() => this.setState({isMenuVisible: true})}
-                                >
-                                    <Icon
-                                        src={Pencil}
-                                        width={variables.iconSizeSmall}
-                                        height={variables.iconSizeSmall}
-                                        fill={themeColors.iconReversed}
-                                    />
-                                </Pressable>
-                                <PopoverMenu
-                                    isVisible={this.state.isMenuVisible}
-                                    onClose={() => this.setState({isMenuVisible: false})}
-                                    onItemSelected={() => this.setState({isMenuVisible: false})}
-                                    menuItems={this.createMenuItems(openPicker)}
-                                    anchorPosition={this.props.anchorPosition}
-                                    animationIn="fadeInDown"
-                                    animationOut="fadeOutUp"
-                                />
+                                {
```

Comment:
Spacing is just a little weird here (should be 4 spaces past the `<>` not 2):

```suggestion
                                  {
```

Reply:
Will check. Not sure how editor didn't pick this.

Reply:
@roryabraham It seems the `eslint` rolls back to two spaces when we use `<>`.  I also checked across the app, it seems that we're using two spaces only after `<>`. 

![image](https://user-images.githubusercontent.com/3069065/132085113-b5607d19-947e-4f61-a6d9-179cb13be230.png)
![image](https://user-images.githubusercontent.com/3069065/132085121-0064d234-e87d-4474-9f5e-fa34ad40900a.png)
![image](https://user-images.githubusercontent.com/3069065/132085126-1423fc8f-33f0-49f5-9e7c-96c379147877.png)

Please let me know if you meant anything else.

Reply:
Okay, no worries about the spacing then, we can just trust ESLint.